components/aboutUs/Resources.tsx

```diff
@@ -306,7 +306,7 @@ export const ABOUT_DATA = [
     title: `What are you using to build this app?`,
     text: (
       <p>
-        It's a <a href="https://nextjs.org/docs">Next.js</a> app powered by a{" "}
+        It&apos;s a <a href="https://nextjs.org/docs">Next.js</a> app powered by a{" "}
```

Comment:
I don't like this change. We should be allowed to write "it's" - otherwise we need a new solution. We shouldn't be forced to use HTML entities - it's not really practical in the long run.

Reply:
Testing this locally, it seems like there's no issues when I change it to "it's". We should change this back :)

Reply:
I believe it is a best practice to use HTML entities within JSX or TSX. We could just forego the Next11 upgrade or backlog it or find a way to customize ESLint so it doesn't check for this. I prefer just to let Next11's recommended linter setting dictate how things go.

Reply:
It makes the code _really_ unreadable if we start adding HTML entities in strings like this, so I doubt it's a best practice 🤔. Maybe for some special cases, but surely not for these cases.

We should never have a problem with encoding, as long as we encode as UTF-8 (which it seems Next does by default).
If it's eslint complaining, we should be able to disable that rule. I would suggest doing that instead.

Reply:
Ok, I've disabled the rule 33fc4731184fbf9908885ce9e2a11d02cbb9c15b

Reply:
Alright. Can you make the same changes to other places where you placed HTML entities? .. I'll resolve this comment when it's done :)